pkg/apis/monitoring/v1beta1/alertmanager_config_types.go

```diff
@@ -1006,6 +1006,10 @@ type PushoverConfig struct {
 	// When true, the message can include HTML formatting tags.
 	// +optional
 	HTML bool `json:"html,omitempty"`
+	// monospace optional HTML/monospace formatting for the message, see https://pushover.net/api#html
+	// html and monospace formatting are mutually exclusive.
+	// +optional
+	Monospace bool `json:"monospace,omitempty"`
```

Comment:
since optional make this as `*bool`?

I see for HTML as well its bool may be we should update that

cc: @simonpasquier